ionicons/IonIcons.m

```diff
@@ -19,10 +19,33 @@ @implementation IonIcons
 + (UIFont*)fontWithSize:(CGFloat)size;
 {
     UIFont* font = [UIFont fontWithName:@"ionicons" size:size];
+    if (!font) {
+        [[self class] dynamicallyLoadFontNamed:@"ionicons"];
```

Comment:
This is really cool. I'm wondering if this method should be in a `dispatch_once` block since it looks like this affects global state instead of returning some artifact, and might be called frequently.